pkgs/build-support/node/build-npm-package/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchNpmDeps, callPackage, nodejs }:
+
+{ name ? "${args.pname}-${args.version}"
```

Comment:
This should probably include `npmBuildScript` and other flags effecting `buildNpmPackage`, just so its a bit clearer what influences the builder without having to read through the hooks.

This would be partially fixed when documentation gets added, but I still see value in having it be clear from the code itself as well. I personally usually end up looking at that rather than the documentation.

For that reason I also think adding comments here as to what the flags do is a good idea.